examples/test.html

```diff
@@ -58,13 +58,18 @@
   attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, &copy; <a href="http://cartodb.com/attributions">CartoDB</a>'
   }).addTo(map);
 
+  var layers = [{
+      sql: 'select * from ne_10m_populated_places_simple_3',
```

Comment:
what's the expectation when we pass an sql string? we're not handling custom sqls in the provider right now